packages/react-chat/src/components/Text/Markdown.tsx

```diff
@@ -35,14 +34,18 @@ const MarkdownText = styled(ReactMarkdown, {
 });
 
 MarkdownText.defaultProps = {
-  rehypePlugins: [rehypeRaw],
   remarkPlugins: [remarkGfm],
-  linkTarget: '_blank',
```

Comment:
this is no longer supported:
https://github.com/remarkjs/react-markdown/pull/761